config/project.ini

```diff
@@ -0,0 +1,251 @@
+[app:main]
```

Comment:
what is this file here, what is it trying to achieve, is it for testing or example?